src/playback/qml/MuseScore/Playback/internal/MixerMuteAndSoloSection.qml

```diff
@@ -66,6 +66,10 @@ MixerPanelSection {
 
                 onToggled: {
                     channelItem.muted = !checked
+
+                    if (channelItem.muted && channelItem.solo) {
```

Comment:
Looks like it should be in MixerChannelItem
The same logic we have for solo in MixerChannelItem::setSolo

Reply:
Ah, of course - fixed!